frontend/server/utils/logger.ts

```diff
@@ -0,0 +1,9 @@
+import { consola } from "consola"
+
+const logger = consola.withTag("Openverse")
+// In production, `info` on the server and silent on the client.
+// In other environments, `debug`.
+logger.level =
+  process.env.NODE_ENV === "production" ? (import.meta.server ? 3 : -999) : 4
+
+export { logger }
```

Comment:
I've noticed a mix of logger and console being used. When are we supposed to use one or the other?

Reply:
Initially, I thought we can't use `logger` from `/server` in the `nuxt` `/src` part of the app, but I forgot about it and logger seems to work in the `/src`, too. I'll try moving the logger to `/shared` top-level folder to show that it's used both by the Nitro server and the Nuxt app.